learn/advanced/filtering_and_faceted_search.md

```diff
@@ -102,10 +102,14 @@ The [`GET` route of the search endpoint](/reference/api/search.md#search-in-an-i
 
 String expressions combine conditions using the following filter operators and parentheses:
 
-- `NOT` only returns documents that do not satisfy a condition : `NOT genres = horror`
+- `NOT` returns all documents that do not satisfy a condition. The expression `NOT genres = horror` returns all documents whose `genres` do not contain `horror` and all documents missing a `genres` field
```

Comment:
We should update the example on line 217. The existing example would return movies that weren't directed by Tim Burton and movies that don't have a `director` field.
Maybe verify the results for the example on line 225 as well

Reply:
I don't see it 👀

Reply:
That's because I forgot to push 😅 

Should be there now.